docs/zephyr_02_functional_requirements.sdoc

```diff
@@ -448,6 +448,448 @@ Zephyr shall provide an interface to assign a specific handler for a fatal error
 
 [/SECTION]
 
+[SECTION]
+TITLE: System Initialization
+
+[REQUIREMENT]
+STATUS: Draft
+TYPE: Functional
+COMPONENT: System Initialization
+TITLE: Initialization follows dependency order
+STATEMENT: >>>
+Zephyr shall initialize the system following the order of component dependencies.
+<<<
+
+[REQUIREMENT]
+STATUS: Draft
+TYPE: Functional
+COMPONENT: System Initialization
+TITLE: Data provided from preceding executable available throughout initialization process
```

Comment:
It could, if there were ever only one requirement around system initialization data. At this point I expect we will find others.

In this case the requirement is specifically around visible lifetime of initialization data from the preceding executable. It does not address initialization data from a factory configuration file.